app/assets/stylesheets/course/forum/topics.scss

```diff
@@ -0,0 +1,13 @@
+.course-forum-topics {
+  &.show {
+    .post {
+      // In case the previous post was too short.
+      clear: both;
+
+      // Spacing between posts.
+      margin-bottom: 1em;
+
+      @include forum-post;
```

Comment:
Expected item on line 10 to appear before line 5. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets